From 4140593b86b3871dfc68a8e4d89d16eac1b4bf4d Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Mon, 27 Feb 2017 16:01:22 +0200
Subject: [PATCH 1435/2241] configs: strongswan ipsec modifications

The configs added to this file are required in order to
work with StrongSwan, more information can be found here:
https://wiki.strongswan.org/projects/strongswan/wiki/KernelModules

Those configs are required for both armv7 and armv8.
Build documentation was updated accordingly.

Change-Id: Ie5c90c35c42b9236049103f9e7673e6710d92ef6
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37099
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 tools/configs/strongswan-ipsec | 15 +++++++++++++++
 1 file changed, 15 insertions(+)
 create mode 100644 tools/configs/strongswan-ipsec

diff --git a/tools/configs/strongswan-ipsec b/tools/configs/strongswan-ipsec
new file mode 100644
index 0000000..1cde52b
--- /dev/null
+++ b/tools/configs/strongswan-ipsec
@@ -0,0 +1,15 @@
+CONFIG_XFRM_USER=y
+CONFIG_NET_KEY=y
+CONFIG_IP_ADVANCED_ROUTER=y
+CONFIG_IP_MULTIPLE_TABLES=y
+CONFIG_INET_AH=y
+CONFIG_INET_ESP=y
+CONFIG_INET_IPCOMP=y
+CONFIG_INET6_AH=y
+CONFIG_INET6_ESP=y
+CONFIG_INET6_IPCOMP=y
+CONFIG_IPV6_MULTIPLE_TABLES=y
+CONFIG_NETFILTER=y
+CONFIG_NETFILTER_XTABLES=y
+CONFIG_NETFILTER_XT_MATCH_POLICY=y
+CONFIG_CRYPTO_ECHAINIV=y
-- 
2.7.4

